sdk/management/samples/src/main/java/com/azure/management/compute/samples/ConvertVirtualMachineToManagedDisks.java

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.management.compute.samples;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.management.Azure;
+import com.azure.management.compute.KnownLinuxVirtualMachineImage;
+import com.azure.management.compute.VirtualMachine;
+import com.azure.management.compute.VirtualMachineSizeTypes;
+import com.azure.management.resources.fluentcore.arm.Region;
+import com.azure.management.resources.fluentcore.profile.AzureProfile;
+import com.azure.management.samples.Utils;
+
+/**
+ * Azure Compute sample for managing virtual machines -
+ *   - Create a virtual machine with un-managed OS and data disks
+ *   - Deallocate the virtual machine
+ *   - Migrate the virtual machine to use managed disk.
+ */
+public final class ConvertVirtualMachineToManagedDisks {
+
+    /**
+     * Main function which runs the actual sample.
+     *
+     * @param azure instance of the azure client
+     * @return true if sample runs successfully
+     */
+    public static boolean runSample(Azure azure) {
+        final String linuxVMName = azure.sdkContext().randomResourceName("VM1", 15);
+        final String rgName = azure.sdkContext().randomResourceName("rgCOMV", 15);
+        final String userName = "tirekicker";
+        // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Serves as an example, not for deployment. Please change when using this in your code.")]
+        final String password = "12NewPA$$w0rd!";
```

Comment:
Please randomize at least part of it (I think Tanyi used the `randomeResourceName` for this purpose. May worth put to Util class if needed in more places).

Reply:
Here https://github.com/Azure/azure-libraries-for-java/pull/863/files. Could add to utils if needed.